mypy/plugins/common.py

```diff
@@ -86,6 +86,9 @@ def add_method(
         return_type: Type,
         self_type: Optional[Type] = None,
         tvar_def: Optional[TypeVarDef] = None,
```

Comment:
Does this mean that you can currently only include a single TypeVar when using this method? Is there a good reason for that limitation?